variables.tf

```diff
@@ -59,6 +59,12 @@ variable "deploy_in_ecs" {
   EOF
 }
 
+variable "superblocks_agent_data_domain" {
```

Comment:
add a validation block in here to ensure that it is either `app.superblocks.com` or `eu.superblocks.com`